src/InstagramClone.Api/Database/InstagramCloneDbContext.cs

```diff
@@ -0,0 +1,41 @@
+﻿using InstagramClone.Api.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api.Database
+{
+    public class InstagramCloneDbContext : DbContext
+    {
+        public InstagramCloneDbContext(): base()
+        {
+
+        }
```

Comment:
your constructor is supposed to have some parameters. Please refer to a documentation to see what parameters you should declare here